### PR TITLE
fix: [CDS-34697]: max height added for text area

### PR DESCRIPTION
**Pull request by samarth-sharma31**
##### Summary:

Max height to be added to text area input

##### Jira Links:

https://harness.atlassian.net/browse/CDS-34697

##### Screenshots:

https://user-images.githubusercontent.com/96409598/157234805-26d504b3-a129-48eb-8093-e1ce43cfb588.mov

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by samarth-sharma31** [comment]
retrigger titlecheck

**Comment by samarth-sharma31** [comment]
retrigger titlecheck

**Comment by samarth-sharma31** [comment]
retrigger titlecheck

File: src/framework/Templates/TemplateConfigModal/TemplateConfigModal.module.scss [hunk 1]
```diff
@@ -17,7 +17,9 @@
   }
 
   .comment {
-    max-height: 240px !important;
+    textarea[name='comment'] {
+      max-height: 240px !important;
+    }
```
Comment (ishantmehtaharness) [inline src/framework/Templates/TemplateConfigModal/TemplateConfigModal.module.scss:22]: Can it be done using className?

Comment (samarth-sharma31) [reply src/framework/Templates/TemplateConfigModal/TemplateConfigModal.module.scss]: Yes, have changed the implementation now. Please review. Thanks!

**Review by ishantmehtaharness — DISMISSED** [review]
Looks good now

**Comment by samarth-sharma31** [comment]
retrigger jest

**Comment by samarth-sharma31** [comment]
retrigger typecheck

**Comment by samarth-sharma31** [comment]
trigger cypress

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Comment by samarth-sharma31** [comment]
trigger cypress

**Review by yc147 — CHANGES_REQUESTED** [review]
Revert changes in src/framework/Templates/TemplateConfigModal/TemplateConfigModal.module.scss. It is not required.

**Review by chiragp1212 — APPROVED** [review]

**Review by yc147 — APPROVED** [review]

**Comment by samarth-sharma31** [comment]
trigger rebase

**Review by kapilkumawat86 — APPROVED** [review]

**Review by yc147 — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Pull request merged**